GoogleForCreators/web-stories-wp
# Updated hover frame sizing and color

## Context

This updates the frame sizing, color and style for all elements including background.

See todo in #6378.

This also removes the hint for the blue drop targets, as this is now covered by the help center.

Finally, this removes the thick blue semitransparent border used when dropping media into shapes or other media.

## User-facing changes

This updates the hover styling, which is also used for "available drop target" highlighting:

| Type | Before | After |
|-|-|-|
| Hover background | ![Screen Shot 2021-02-15 at 00 28 14](https://user-images.githubusercontent.com/637548/107908803-e10a8180-6f24-11eb-99cd-a32485c68ed0.png) | ![Screen Shot 2021-02-15 at 00 26 40](https://user-images.githubusercontent.com/637548/107908811-e4057200-6f24-11eb-9894-7e90d50b562c.png) |
| Hover shape | ![Screen Shot 2021-02-15 at 00 27 59](https://user-images.githubusercontent.com/637548/107908926-275fe080-6f25-11eb-87b7-03e6b2c165c9.png) | ![Screen Shot 2021-02-15 at 00 26 44](https://user-images.githubusercontent.com/637548/107908817-e798f900-6f24-11eb-8127-5ee0d7cefb74.png) |
| Hover text | ![Screen Shot 2021-02-15 at 00 28 03](https://user-images.githubusercontent.com/637548/107908834-f1baf780-6f24-11eb-996a-29289968d553.png) | ![Screen Shot 2021-02-15 at 00 26 48](https://user-images.githubusercontent.com/637548/107908840-f4b5e800-6f24-11eb-893e-d1e073fb6fd6.png) |
| Hover media | ![Screen Shot 2021-02-15 at 00 28 09](https://user-images.githubusercontent.com/637548/107908849-f8e20580-6f24-11eb-8cbe-fe872521ecf9.png) | ![Screen Shot 2021-02-15 at 00 26 51](https://user-images.githubusercontent.com/637548/107908854-fc758c80-6f24-11eb-8e6f-69ab8152ffb9.png) |
| Show drop targets | ![Screen Shot 2021-02-15 at 00 28 26](https://user-images.githubusercontent.com/637548/107908862-00091380-6f25-11eb-883a-794cb8c2ba1d.png) | ![Screen Shot 2021-02-15 at 00 27 33](https://user-images.githubusercontent.com/637548/107908868-026b6d80-6f25-11eb-8987-b1ccbd9ae0f3.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Add a shape, text and media element to page
2. Hover each of the elements and the white background element and observe the hover outline - it's thin, light blue and fades in and out with .5s transition
3. Start draggin an image from the library
4. Observe that the droppable element (background, shape and media) shows the same outline as on hover) and that text does not.

### UAT






## Checklist




---

Fixes #6378
